lib/urlwatch/filters.py

```diff
@@ -130,6 +132,9 @@ def __init__(self, job, state):
         super().__init__(job, state)
 
         self.hooks = None
+        if not os.path.exists(self.FILENAME):
+            self.FILENAME = os.path.join(AppDirs('urlwatch').user_config_dir, 'lib/hooks.py')
```

Comment:
I meant:

    self.FILENAME = os.path.join(AppDirs('urlwatch').user_config_dir, 'lib', 'hooks.py')

Reply:
Or actually, wait -- legacy hooks would only be in the older folder, so there's no need to change anything here.

Reply:
Meaning if it doesn't exist, don't try to find it in the XDG-based directory.

Reply:
Reverted it all to the original.